chain/consensus/filcns/upgrades.go

```diff
@@ -192,13 +192,18 @@ func DefaultUpgradeSchedule() stmgr.UpgradeSchedule {
 		}},
 		Expensive: true,
 	}, {
-		Height:    build.UpgradeV17Height,
+		Height:    build.UpgradeSharkHeight,
 		Network:   network.Version17,
 		Migration: UpgradeActorsV9,
 		PreMigrations: []stmgr.PreMigration{{
 			PreMigration:    PreUpgradeActorsV9,
-			StartWithin:     180,
+			StartWithin:     240,
```

Comment:
Do we have numbers for premigrations? I'm assuming that we do, but want to sanity check

Reply:
Yeah, we're looking at roughly 40 mins for the premigration, but we _really_ want this to finish, so 2 hours is the window I'm going with.